benchmarks/benchmarks.py

```diff
@@ -64,49 +77,58 @@ def time_startswith(self):
         self.df["str"].str.startswith("10")
 
     def time_startswith_ext(self):
-        self.df_ext["str"].text.startswith("10")
+        self.df_ext["str"].str.startswith("10")
+
+    def time_startswith_ext_fr(self):
+        self.df_ext["str"].fr_str.startswith("10")
 
     def time_endswith(self):
         self.df["str"].str.endswith("10")
 
     def time_endswith_ext(self):
-        self.df_ext["str"].text.endswith("10")
+        self.df_ext["str"].fr_str.endswith("10")
+
+    def time_endswith_ext_fr(self):
+        self.df_ext["str"].str.endswith("10")
 
     def time_cat(self):
         self.df["str"].str.cat(self.df["str"])
 
     def time_cat_ext(self):
-        self.df_ext["str"].text.cat(self.df_ext["str"])
+        self.df_ext["str"].fr_str.cat(self.df_ext["str"])
 
     def time_zfill(self):
         self.df["str"].str.zfill(10)
 
     def time_zfill_ext(self):
-        self.df_ext["str"].text.zfill(10)
+        self.df_ext["str"].fr_str.zfill(10)
 
     def time_contains_no_regex(self):
         self.df["str"].str.contains("1012102", regex=False)
 
     def time_contains_no_regex_ext(self):
-        self.df_ext["str"].text.contains("1012102", regex=False)
+        self.df_ext["str"].str.contains("1012102", regex=False)
```

Comment:
```suggestion
        self.df_ext["str"].fr_str.contains("1012102", regex=False)
```

Reply:
This one is actually correct, it should be `str`

Reply:
got it